microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1777,6 +1778,47 @@ See also:
 ---
 
 
+<!-- ------------------------------ -->
+#### WebResourceRequested for workers - INCOMING
```

Comment:
Where to put these links? Options eg:
* Add to the above section w/ no notes.
* New h6 section here, w/ intro note.
* Other design options...
@peiche-jessica @victorhuangwq

Reply:
I'd merge it to "Manage network requests in WebView2" since this is expanding the existing `WebResourceRequested`  events to workers scenarios.